contracts/lib/Approvable.sol

```diff
@@ -3,18 +3,19 @@ pragma solidity ^0.4.11;
 import './Ownable.sol';
 
 contract Approvable is Ownable {
-  mapping(address => bool) approved;
+  mapping(address => bool) public approved;
 
   function Approvable() {
-    approveAddress(msg.sender);
+    updateAddressState(msg.sender, true);
```

Comment:
Just inline adding the sender:
```
approved[msg.sender] = true;
```

Or if that feels weird you could break out an `internal` function which does the actual work (without the `onlyApproved` check) and have `updateAddressState` call that internal function.

Not sure if the latter works.